ckanext/switzerland/helpers.py

```diff
@@ -9,7 +9,7 @@
 from babel import numbers
 import iribaker
 from urlparse import urlparse
-from ckan.lib.helpers import localised_number, url_for
+import ckan.lib.helpers as ckan_helpers
```

Comment:
We're still only using three helper functions, so I would just add `lang` to the list that we import rather than importing the whole library.